website/docs/patterns/table-multi-select/partials/interaction-concepts/interaction-concepts.md

```diff
@@ -39,7 +39,7 @@ If not handled in a clear and logical manner, how the selection is persisted acr
 
 ### Within a paginated Table
 
-If a user makes a selection on one page of a paginated Table and progresses to the next page, persisting this selection can be communicated by the [selected count](/patterns/multi-select-patterns?tab=guidelines#selected-count) and can be helpful in scenarios where much of the data is similar or overlaps.
+If a user makes a selection on one page of a paginated Table and progresses to the next page, persisting this selection can be communicated by the [selected count](/patterns/table-multi-select?tab=guidelines#selected-count) and can be helpful in scenarios where much of the data is similar or overlaps.
```

Comment:
I'm not sure if you need the tab URL param here, when I clicked I was navigated to the top of the page. This _might_ solve that, but double check.

```suggestion
If a user makes a selection on one page of a paginated Table and progresses to the next page, persisting this selection can be communicated by the [selected count](/patterns/table-multi-select#selected-count) and can be helpful in scenarios where much of the data is similar or overlaps.
```

Reply:
Additional code would be needed to use anchors within Tabs. This would be a nice feature add if the team thinks it is worth putting on the board, which is why I left it in the link. I will bring this up in our next sync.

Reply:
good catch! Yup, just totally removed it from link so anchor scrolling is fixed as well.